src/dotnet-install.sh

```diff
@@ -1481,6 +1481,10 @@ say "- The SDK needs to be installed without user interaction and without admin
 say "- The SDK installation doesn't need to persist across multiple CI runs."
 say "To set up a development environment or to run apps, use installers rather than this script. Visit https://dotnet.microsoft.com/download to get the installer.\n"
 
+if [ "$internal" = true ] && [ -z "$feed_credential" ]; then
+    say_warning "Note, it is necessary to use --feed-credential option when --internal switch is set."
```

Comment:
We have `--` before the switches but not in powershell. Should we make them the same in both scripts?

Reply:
Probably not because 1) powershell code won't work with --Internal, and 2) -internal in sh would mean that `-i -n -t -e -r -n -a -l` parameters are used in sh?

Reply:
I suppose @bozturkMSFT means that I missed "-" before the switch name in the similar message in the ps script. I will add it.